src/main/java/application/FormController.java

```diff
@@ -143,6 +156,8 @@ public SubmitResponseBean submitForm(@RequestBody SubmitRequestBean submitReques
                 formEntrySession.getFormEntryModel(),
                 submitRequestBean.getAnswers());
 
+        FormVolatilityRecord volatilityRecord = formEntrySession.getSessionVolatilityRecord();
```

Comment:
Nit: would be good to move this down to where it ends up getting used. (above L213)